lib/src/utils/screen_util.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:flutter_device_type/flutter_device_type.dart';
+
+class ScreenUtil {
+  double height(double designedPixel) {
```

Comment:
We need some variables to tell what these numbers are.

Reply:
added screen variables